city/schemas.py

```diff
@@ -0,0 +1,17 @@
+from pydantic import BaseModel
+
+
+class CityBase(BaseModel):
+    name: str
+    additional_info: str
+
+    class Config:
+        orm_mode = True
+
```

Comment:
(https://docs.pydantic.dev/latest/migration/#changes-to-config)

The following config settings have been renamed:

orm_mode → from_attributes